api.yaml

```diff
@@ -322,7 +322,65 @@ paths:
               #! end !#
               "bodyPassthrough": true
             }
-
+  /v1/cloud/arnid/{arnid}:
+    get:
+      summary: "Retrieve a cloud asset at a point in time by ARN ID"
+      parameters:
+        - name: "arnid"
+          in: "path"
+          description: "The arn id of the asset"
+          required: true
+          schema:
+            type: "string"
+        - name: "time"
+          in: "query"
+          description: "The point in time details for a given asset"
+          required: true
+          schema:
+            type: "string"
+            format: "date-time"
```

Comment:
It seems like implementation is using RFC3339Nano which is very specific time formatting. Do we expect consumers of API to understand this formatting? If so, i suggest we communicate with them in more details.

Reply:
We generally tend to use the api.yaml when looking up how to make requests to endpoints, a comment here indicating this time is in `RFC3339Nano` form would be helpful as a start. Eventually this information should exposed in the api docs for consumers to be aware of.

Reply:
@sydneyteh96 could be useful to add a comment that the date-time is `RFC3339Nano` format but I'll leave it up to you, it could technically be done in the API docs.

Reply:
Yes, I can add a comment.